src/pages/iou/MoneyRequestTagPage.js

```diff
@@ -109,6 +109,14 @@ export default compose(
             key: ONYXKEYS.IOU,
         },
     }),
+    //  eslint-disable-next-line
+    withOnyx({
+        report: {
+            // Fetch report ID from IOU participants if no report ID is set in route
+            key: ({route, iou}) => `${ONYXKEYS.COLLECTION.REPORT}${lodashGet(route, 'params.reportID', '') || lodashGet(iou, 'participants.0.reportID', '')}`,
+        },
+    }),
```

Comment:
@studentofcoding This change is not necessary, right? It was removed here https://github.com/Expensify/App/commit/6a47b41dbfb39ed6cd46ba0493aaca7a00888ac6

Reply:
Ahh yes, I missed it and added it again